test/integration/skills.test.js

```diff
@@ -0,0 +1,93 @@
+const chai = require("chai");
+const { expect } = chai;
+const chaiHttp = require("chai-http");
+
+const app = require("../../server");
+const authService = require("../../services/authService");
+const addUser = require("../utils/addUser");
+const { fetchUser } = require("../../models/users");
+const skillsData = require("../fixtures/skills/skills")();
+const cleanDb = require("../utils/cleanDb");
+
+const config = require("config");
+const cookieName = config.get("userToken.cookieName");
+
+chai.use(chaiHttp);
+
+describe("Skills", function () {
+  let jwt;
+  let userName;
+
+  beforeEach(async function () {
+    const userId = await addUser();
+    jwt = authService.generateAuthToken({ userId });
+
+    const { user } = await fetchUser({ userId });
+    userName = user.username;
+  });
+
+  afterEach(async function () {
+    await cleanDb();
+  });
+  describe("POST /skills/:username", function () {
```

Comment:
More cases need to be covered here like when username is not found, when there's no user found with the username,etc